### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by wid2001** [igorescobar/jQuery-Mask-Plugin#618]
Thanks for the outstanding work.  Your does exactly what I am trying to accomplish.  I apologize for the inexperience and the lack of knowledge but after much research I have not been able to figure out how to implement this plugin.  
I am attempting to install on a webform that I am utilizing.  I have uploaded the jquery to my pardot site and linked it externally on my form.

`<!DOCTYPE html>
<html>
	<head>
		<base href="" >
		<meta charset="utf-8"/>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
		<meta name="description" content="%%description%%"/>
		<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js"></script>
		<script type="text/javascript" src="http://info.mission43.org/l/427472/2017-10-17/7x4q6c/427472/271664/jquery.mask.min.js"></script> 
<script>		
    $(document).ready(function(){
    $('#427472_443981pi_427472_443981').mask('(000) 000-0000');
});</script>`

I have tried using both the ID (pictured abbove) and the class and have not been able to see any results.  

The form I am using is locate [here](http://info.mission43.org/l/427472/2017-10-15/7wrpbz)
Would you mind pointing me in the right direction?  I am sure the problem comes from my lack of knowledge and experience but I thought you might be able to point me in the right direction.

Thanks

Spence

**Comment by igorescobar** [comment]
@wid2001 please provide a functional jsfiddle. follow the issues guidelines, please.
